style.css

```diff
@@ -1,542 +1,766 @@
+  font-weight: 700;
   color: #fff;
-  margin-bottom: 50px;
 }
 
-.contact-me p {
-  font-weight: 400;
-  font-style: normal;
+.secondary-text {
   font-size: 20px;
   line-height: 28px;
-  text-align: center;
+  font-weight: 400;
   color: #ebebff;
-  width: 327px;
-  height: 102px;
-  margin: 12px 24px -30px 24px;
+  text-align: center;
 }
 
-#name {
-  display: block;
-  box-sizing: border-box;
-  min-width: 327px;
-  height: 48px;
-  border-radius: 8px;
-  background-color: white;
-  border: 1px solid #cfd8dc;
-  margin-right: 24px;
-  margin-left: 24px;
-  margin-top: 87px;
-  padding-left: 16px;
-  padding-right: 16px;
-  font-weight: 400;
-  font-style: normal;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
+.contact-bg {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  transform: translateY(-50%);
+  height: 95%;
 }
 
-#email {
-  padding-right: 16px;
-  font-weight: 400;
-  box-sizing: border-box;
-  font-style: normal;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  display: block;
-  width: 327px;
-  height: 48px;
-  border-radius: 8px;
-  background-color: white;
-  border: 1px solid #cfd8dc;
-  margin-right: 24px;
-  margin-left: 24px;
-  margin-top: 20px;
-  padding-left: 16px;
-}
-
-#msg {
-  width: 327px;
-  box-sizing: border-box;
-  height: 170px;
-  margin-left: 24px;
-  margin-right: 24px;
-  margin-top: 18px;
-  border-radius: 8px;
-  border: 1px solid #cfd8dc;
-  padding: 15px 38px 15px 16px;
-  font-weight: 400;
-  font-style: normal;
-  font-size: 15px;
-  line-height: 24px;
+.contact-form {
+  width: 100%;
+}
+
+.contact-form ul {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  list-style: none;
+  gap: 2rem;
 }
 
-[placeholder*="Write"] {
-  color: #b3bac5;
+/* Alert message start */
+
+.show-msg {
+  color: crimson;
+  background-color: #fff;
+  border-radius: 0.6rem;
+  display: none;
+  font-size: 1.3rem;
+  padding: 0.5rem;
+  text-align: center;
+  animation: show-msg 1s 1;
+  margin-top: -2rem;
+  margin-bottom: 0.4rem;
 }
 
-.homepage-indicator {
-  position: relative;
-  width: 134px;
-  height: 5px;
-  border-radius: 10px;
-  color: #0d151e;
-  bottom: -35px;
-  left: -50px;
+@keyframes show-msg {
+  from {
+    visibility: hidden;
+    background-color: #fff;
+  }
+
+  to {
+    border-radius: 1rem;
+  }
 }
 
-.form-button {
-  width: 140px;
-  height: 48px;
-  color: #6070ff;
-  padding: 12px 16px;
-  position: static;
-  left: 24px;
-  font-weight: 500;
-  font-size: 17px;
-  font-style: normal;
-  margin-left: 24px;
-  margin-top: 22px;
-  text-align: center;
+/* Alert message end */
+
+.contact-element {
+  border: 0.1rem solid #cfd8dc;
+  padding: 1.5rem;
+  border-radius: 0.8rem;
+  width: 100%;
+  outline: none;
+  font-size: 1.4rem;
 }
 
-@media only screen and (max-width: 480px) {
-  .contact-me {
+.contact-element:focus::placeholder {
+  color: #3a3c3f;
+}
+
+.btn-contact {
+  margin: 12px 90px;
+  background-color: #fff;
+}
```

Comment:
[OPTIONAL]
- I would recommend centering the submit button to improve the UI design.
<img width="470" alt="image" src="https://user-images.githubusercontent.com/77190203/150387162-0a46d252-5114-40ea-b42b-c8aad5a408c2.png">

Reply:
Ok I will centering the submit button.